Gems/MiniAudio/Code/Source/Clients/MiniAudioPlaybackComponentController.cpp

```diff
@@ -168,6 +263,19 @@ namespace MiniAudio
         if (m_sound)
         {
             ma_sound_set_position(m_sound.get(), world.GetTranslation().GetX(), world.GetTranslation().GetY(), world.GetTranslation().GetZ());
+
+            // Set the forward direction for the sound source
+            AZ::Transform worldTm = AZ::Transform::CreateIdentity();
+            AZ::TransformBus::EventResult(worldTm, m_entityComponentIdPair.GetEntityId(), &AZ::TransformBus::Events::GetWorldTM);
+            const AZ::Vector3 forward = worldTm.TransformVector(AZ::Vector3::CreateAxisY(1.f));
+            if (!m_config.m_fixedDirection)
+            {
+                ma_sound_set_direction(m_sound.get(), forward.GetX(), forward.GetY(), forward.GetZ());
+            }
+            else
+            {
+                ma_sound_set_direction(m_sound.get(), m_config.m_direction.GetX(), m_config.m_direction.GetY(), m_config.m_direction.GetZ());
+            }
```

Comment:
lets avoid getting the direction when the sound direction is fixed.  Also you should be able to use the `GetBasisXYZ` functions to get the unit XYZ vectors of a transform.
```suggestion
            if (!m_config.m_fixedDirection)
            {
                AZ::Transform worldTm = AZ::Transform::CreateIdentity();
                AZ::TransformBus::EventResult(worldTm, m_entityComponentIdPair.GetEntityId(), &AZ::TransformBus::Events::GetWorldTM); 
                ma_sound_set_direction(m_sound.get(), worldTM.GetBasisX(), worldTM.GetBasisY(), worldTM.GetBasisZ()));
            }
            else
            {
                ma_sound_set_direction(m_sound.get(), m_config.m_direction.GetX(), m_config.m_direction.GetY(), m_config.m_direction.GetZ());
            }
```

Reply:
Looks like `GetBasisXYZ` returns a Vector3:
```
AZ_MATH_INLINE Vector3 Transform::GetBasisY() const
{
    return m_rotation.TransformVector(Vector3::CreateAxisY(m_scale));
}
```

Though since for the forward direction we're only doing this for the entity's +Y direction I may be able to just write it out as `worldTm.TransformVector(AZ::Vector3::CreateAxisY(1.f)).GetX(), worldTm.TransformVector(AZ::Vector3::CreateAxisY(1.f)).GetY(), worldTm.TransformVector(AZ::Vector3::CreateAxisY(1.f)).GetZ()`.

Reply:
You're right, in this case you should just be able to use: `ma_sound_set_direction(m_sound.get(), worldTM.GetBasisY().GetX(), worldTM.GetBasisY().GetY(), worldTM.GetBasisY().GetZ()));
We're just trying to avoid any unnecessary calculations if `GetBasisY()` returns the same thing as `worldTm.TransformVector(AZ::Vector3::CreateAxisY(1.f))`

Reply:
@AMZN-alexpete I'm having it use GetBasisY now: cce15c639e507882a547db1b612cffe29dd92b62